logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/GetNotificationsUseCase.kt

```diff
@@ -254,16 +254,16 @@ internal class GetNotificationsUseCaseImpl internal constructor(
         is MessageContent.Availability -> false
         is MessageContent.FailedDecryption -> false
         is MessageContent.MissedCall -> true
-        is MessageContent.Empty -> false
         is MessageContent.Ignored -> false
         is MessageContent.LastRead -> false
         is MessageContent.Cleared -> false
         is MessageContent.ConversationRenamed -> false
         is MessageContent.TeamMemberRemoved -> false
+        is MessageContent.Receipt -> false
     }
 
     private fun shouldMessageBeVisibleAsNotification(message: Message) =
-        message.visibility == Message.Visibility.VISIBLE
+        message is Message.Standalone && message.visibility == Message.Visibility.VISIBLE
```

Comment:
Reactions aren't displayed as  in notifications in AR?

Reply:
At the moment? No. It's intended per design. But I'm not sure for how long.

I think this part of the notifications logic should change to query messages and reactions as different sources.

At the moment it's expected that they'd come from the same place, but I'm but sure this should be the case.